@types/index.d.ts

```diff
@@ -145,7 +145,7 @@ export interface Body extends Pick<BodyMixin, keyof BodyMixin> {}
 
 export type RequestRedirect = 'error' | 'follow' | 'manual';
 export type ReferrerPolicy = '' | 'no-referrer' | 'no-referrer-when-downgrade' | 'same-origin' | 'origin' | 'strict-origin' | 'origin-when-cross-origin' | 'strict-origin-when-cross-origin' | 'unsafe-url';
-export type RequestInfo = string | Request;
+export type RequestInfo = string | URL | Request;
```

Comment:
The official TypeScript types have added `URL` as an acceptable parameter to `fetch`, but `RequestInfo` still doesn't include `URL`. 

https://github.com/microsoft/TypeScript/blob/f43cd0accac3e5033820fd27035731ed88f54938/lib/lib.dom.d.ts#L18281

Is there a reason why we are doing it differently here?

Reply:
That's a good question. It seems that in `lib.dom.d.ts` it's always used like this: `RequestInfo | URL`, except for `Cache.addAll`. However, I just tried in Chrome and it seems to accept `URL` just fine as well. So moving `URL` into `RequestInfo` seems like a pretty good optimization to me, I'll be opening such PR to TS shortly.

Reply:
Ah, never mind, seems like `| URL` is being added conditionally here: https://github.com/microsoft/TypeScript-DOM-lib-generator/blob/dbe454edfe3804b0bbfa1ee1a0a88e9c0a1d4092/src/build/emitter.ts#L679
So I can't see an easy fix, even though semantically `RequestInfo = string | URL | Request` probably would make more sense.
Probably better to stick to what they have to avoid conflicts, updating now.

Reply:
Also added `URL` to `Request` constructor, as in TS: https://github.com/microsoft/TypeScript/blob/f43cd0accac3e5033820fd27035731ed88f54938/lib/lib.dom.d.ts#L11733

Reply:
Awesome! 🎉